cli/src/asset_identifier.rs

```diff
@@ -0,0 +1,66 @@
+use error::InvalidInputError;
+use stellar_client::resources::AssetIdentifier;
+
+pub fn parse_identifier_from_str(s: &str) -> Result<AssetIdentifier, InvalidInputError> {
+    let asset_parts: Vec<&str> = s.split("-").collect();
```

Comment:
use `splitn` so that we don't split the issuer.

Reply:
Won't work if they put too many `-`'s in as splitn returns the rest of the string once it hit's the nth occurrence.  We could still check the last string to see if there is an extra `-` but no gain over current implementation and I like this one.  But thanks for pointing out splitn.  For sure good to know about.